### PR TITLE
Don't require libsmbclient as it's pulled in by libsmbclient-devel; (bsc#972197).

### PR DESCRIPTION
**Pull request by lmuelle**

### CODE REVIEW

File: package/yast2-samba-server.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Mar 23 17:55:31 UTC 2016 - lmuelle@suse.com
+
+- Don't require libsmbclient as it's pulled in by libsmbclient-devel;
```
Comment (lslezak) [inline package/yast2-samba-server.changes]: I guess the trailing `;` is a typo.

**Comment by ddiss** [comment]
@lmuelle Please squash both commits into a single change. Otherwise this looks fine:
Reviewed-by: David Disseldorp <ddiss@suse.de>

File: package/yast2-samba-server.changes (hunk 1 above)
Comment (ddiss) [inline package/yast2-samba-server.changes]: A semicolon is commonly used as a separator between changelog descriptions and bug numbers.

**Comment by lmuelle** [comment]
Squashed into a single change.

**Comment by ddiss** [comment]
LGTM. Thanks Lars!

**Pull request merged**